cluster/install/fluo-cluster/bin/impl/install.sh

```diff
@@ -141,7 +152,15 @@ DONE
   done
 }
 
-for service in "$@"; do
+SERVICES=$@
+if [ "$SERVICES" == "--use-config" ]; then
+  host=`hostname`
+  line=`grep $host $CONF_DIR/hosts/hosts_with_services`
```

Comment:
what if grep does not find anything?  thinking of case where what hostname returns does not match whats in file